contrib/ansible/roles/skydive_dev/tasks/main.yml

```diff
@@ -49,7 +49,7 @@
      - selinux-policy-devel
      - policycoreutils-devel
      - mock
-     - yum-utils
+     - dnf-utils
```

Comment:
I'm not sure it would work on centos / rhel ?

Reply:
This is the `skydive_dev` role. We use for our CI machines and the vagrant boxes so we don't need to bother about CentOS and RHEL